subsys/usb/class/netusb/function_eem.c

```diff
@@ -173,24 +172,17 @@ static void eem_read_cb(u8_t ep, int size, void *priv)
 			break;
 		}
 
-		pkt = net_pkt_get_reserve_rx(K_FOREVER);
+		pkt = net_pkt_alloc_with_buffer(netusb_net_iface(),
+						eem_size - 4, AF_UNSPEC, 0,
+						K_FOREVER);
 		if (!pkt) {
 			LOG_ERR("Unable to alloc pkt\n");
 			break;
 		}
 
-		frag = net_pkt_get_frag(pkt, K_FOREVER);
```

Comment:
With this change, the declaration `struct net_buf *frag;` above can be removed, otherwise the compiler emits a warning.